released/packages/coq-monae/coq-monae.0.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "reynald.affeldt@aist.go.jp"
+homepage: "https://github.com/affeldt-aist/monae"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+license: "GPL-3.0-or-later"
+authors: [
+  "Reynald Affeldt"
+  "David Nowak"
+  "Takafumi Saikawa"
+  "Jacques Garrigue"
+  "Celestine Sauvage"
+  "Kazunari Tanaka"
+]
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" { >= "8.10" & < "8.11~" }
```

Comment:
```suggestion
  "coq" { >= "8.10" & < "8.12~" }
```
Works fine for on Coq 8.11.0.

Reply:
Thanks for the information. This will be useful. I updated the opam file.